package.json

```diff
@@ -123,7 +122,6 @@
     "@types/jest-axe": "^3.5.9",
     "@types/jsonwebtoken": "^9.0.6",
     "@types/jwk-to-pem": "^2.0.3",
-    "@types/k6": "^0.52.0",
```

Comment:
Hm, I could swear this was used by `./src/scripts/loadtest/hibp.js`... maybe only for VSCode? I guess as long as it's passing lint and looks OK in VSCode then sure.

Reply:
Ah no, you're right, I think that might have been because Knip didn't see a direct import from `k6`? Reverted in 3674b06d1.

I got too many false positives with Knip anyway, so I've shelved it for now. Might re-investigate after everything is TS, perhaps that helps.